networkx/algorithms/__init__.py

```diff
@@ -11,6 +11,7 @@
 from networkx.algorithms.cycles import *
 from networkx.algorithms.cuts import *
 from networkx.algorithms.dag import *
+from networkx.algorithms.disjoint_paths import *
```

Comment:
Nevermind, I just found it. I browsed a bit more carefully through the build log and saw it performed an actual setup of networkx. I had to add the new module name to setup.py...